designer/server/src/models/forms/form-lifecycle.js

```diff
@@ -11,7 +11,7 @@ export function confirmationPageViewModel(form, errorList) {
   const pageTitle = 'Are you sure you want to make the draft live?'
 
   const formPath = formOverviewPath(form.slug)
-  const navigation = getFormSpecificNavigation(formPath)
+  const navigation = getFormSpecificNavigation(formPath, form, '')
```

Comment:
Do we need this if `''` is the default?

```suggestion
  const navigation = getFormSpecificNavigation(formPath, form)
```

Reply:
Good spot 👍

Reply:
Pushed.